### PR TITLE
Addition of the attribute 'description' to a few applicable resources

### PR DESCRIPTION
**Pull request by shrsr**

### CODE REVIEW

**Review by sajagana — CHANGES_REQUESTED** [review]
The current changes looks good, the `description` field should be updated in the data source and resource documentation files.

File: vendor/github.com/ciscoecosystem/mso-go-client/models/schema_template_contract_filter.go [hunk 1]
```diff
@@ -1,9 +1,10 @@
 package models
 
-func NewTemplateContractFilterRelationShip(ops, path, action, priority string, filterRef map[string]interface{}, directives []interface{}) *PatchPayload {
+func NewTemplateContractFilterRelationShip(ops, path, action, priority, desc string, filterRef map[string]interface{}, directives []interface{}) *PatchPayload {
```
Comment (akinross) [inline vendor/github.com/ciscoecosystem/mso-go-client/models/schema_template_contract_filter.go:3]: I think this one is missing description attribute in the docs

**Review by akinross — CHANGES_REQUESTED** [review]

**Review by akinross — COMMENTED** [review]
will you update the mso-go-client vendor files also?

**Review by akinross — DISMISSED** [review]
LGTM

**Review by akinross — APPROVED** [review]
LGTM

**Review by lhercot — APPROVED** [review]
LGTM

**Pull request merged**